hw/top_earlgrey/ip/sensor_ctrl/doc/checklist.md

```diff
@@ -0,0 +1,274 @@
+---
+title: "SENSOR_CTRL Checklist"
+---
+
+<!--
+NOTE: This is a template checklist document that is required to be copied over to the 'doc'
+directory for a new design that transitions from L0 (Specification) to L1 (Development)
+stage, and updated as needed. Once done, please remove this comment before checking it in.
+-->
```

Comment:
you may want to remove this comment here

Reply:
ah yes.